src/contribute-to-tidb/report-an-issue.md

```diff
@@ -0,0 +1,35 @@
+
+## Filing an issue
+
+If the problem you’re reporting is not already in the issue tracker, you can [open a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) with your GitHub account. TiDB uses issue template for different kinds of issues. Issue templates are a bundle of questions to collect necessary information about the problem so it's easy for other contributors to participate. For example a bug report issue template consists of four questions:
+
+  - Minimal reproduce step.
+  - What did you expect to see?
+  - What did you see instead?
+  - What is your TiDB version?
+
+Answering these questions give the details about your problem so other contributors or TiDB users could pick up your issue more easily. 
+
+As previous section shows, duplicated issues should be reduced. To help others who encountered the problem find your issue, except for problem details answered in the issue template, a descriptive title which contains information that might be unique to it also helps. This can be the components related to or database feature used, the conditions that trigger the bug, or part of the error message if there is any. 
+
+## Understanding the issue’s progress and status
+
+Once your issue is created, other contributors might take part in. You need to discuss with them, provide more information they might want to know to reach consensus and make the progress proceeds. But please realize there are always more pending issues than contributors are able to handle, and especially TiDB community is a global one, contributors reside all over the world and they might already be very busy with their own work and life. Please be patient! If your issue gets stale for some time, it's okay to ping the reviewers and asignees, or take it to [internal.tidb.io](https://internals.tidb.io) for more attention.
+
+## Disagreement with a resolution on the issue tracker
+
+As humans, we will have differences of opinions from time to time. First and foremost, please be respectful that care, thought, and volunteer time went into the resolution.
+With this in mind, take some time to consider any comments made in association with the resolution of the issue. On reflection, the resolution steps may seem more reasonable than you initially thought.
+If you still feel the resolution is incorrect, then raise a thoughtful question on [internal.tidb.io](https://internals.tidb.io). Further argument and disrespectful discourse on [internal.tidb.io](https://internals.tidb.io) after a consensus has been reached amongst the committers is unlikely to win any converts.
```

Comment:
What is disrespectful discourse...?

Reply:
What do you mean? It's just "disrespectful", like insults and aggressive words.

Reply:
Sorry I misunderstood the sentence

Reply:
btw I think the last sentence may be redundant. 
```suggestion
If you still feel the resolution is incorrect, then raise a thoughtful question on [internal.tidb.io](https://internals.tidb.io) for wider attention and discussion.
```

Reply:
Actually this section is not from my hand, I adapted from the CPython development guide. As I understand, this sentence is to persuade some contributors to respect the committers' decision, and not keep pushing their own ideas even after there is a consensus among committers. For example when the author's opinion is different from the committers'. From my own experience in Python community, this is an important part and something you must obey to get promoted.
I don't really insist on this sentence.

Reply:
ok, lgtm. Just thought it a little bit wordy for the case of tidb community.